.github/workflows/release.yml

```
@@ -103,7 +103,8 @@ jobs:
           do
             archive=$(echo ${archive} | sed -e "s/\/$//")
             cp -R shell-completions ${archive}/completions
-            cp cli/cli/src/install.sh ${archive}/install.sh
+            cp cli/scripts/install.sh ${archive}/install.sh
+            cp cli/scripts/phylum-uninstall.sh ${archive}/phylum-uninstall.sh
```

```suggestion
            cp cli/scripts/uninstall-phylum.sh ${archive}/uninstall-phylum.sh
```
Suggestion is for tab completion...b/c typing `ph<tab>` or `phylum<tab>` should show the CLI binary options and not other executables.

There is a case that could be made...for another issue...to provide the uninstall feature within the CLI binary itself instead of a standalone script. I've seen other tools do this...and then they only add a single line/entry to the *rc conf files.

---

Hm I think worrying about completions at all is probably an indication that it's not a great idea to have this in the `$PATH` to begin with?

> There is a case that could be made...for another issue...to provide the uninstall feature within the CLI binary itself instead of a standalone script. I've seen other tools do this...and then they only add a single line/entry to the *rc conf files.

I've considered this but I don't think that's the best solution considering users already interact with an `inshall.sh`, so it's kinda natural that the archive would also come with an `uninstall.sh` rather than putting that inside `phylum` directly.

---

`install.sh` is going to go through some changes and users may not directly interact with it for very long. The end goal is something like:

```
curl http://get.phylum.io | sh -
```

See [this user story](https://www.notion.so/phylum/Install-Script-Improvement-880c22380d4041b388b7937a603ad6aa) for some of the details.

Plus, I think `phylum uninstall` matches well with the current `phylum update` command.